web/src/main/webResources/WEB-INF/config-spring-geonetwork.xml

```diff
@@ -32,10 +32,10 @@
         http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop.xsd">
   <context:annotation-config/>
 
-  <context:property-placeholder location="${app.properties},file://#{systemEnvironment['geonetwork_dir']}/config/app.properties"
```

Comment:
You should keep this - it comes from https://github.com/geonetwork/core-geonetwork/pull/6954.
Same below for the migration_onstartup